### PR TITLE
Update to 21w17a

### PR DESCRIPTION
**Pull request by IMS212**
A new branch should be created for this and the code should be looked over/cleaned up before merge

Notable change: Hardcoded glow squid brightness to 15 (Should this be changed?)

Most notable issue is stuttering/FPS drops when using a handheld dynamic light.

I have not encountered any bugs in my short testing, however more extensive testing should be done before merge.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -41,7 +41,11 @@ repositories {
     }
     maven {
         name = "Cotton"
-        url = "http://server.bbkr.space:8081/artifactory/libs-release/"
+        url = "https://server.bbkr.space:8081/artifactory/libs-release/"
```
Comment (ImperatorStorm) [inline build.gradle:44]: this website doesn't utilize https, you're going to need to explicitly set `allowInsecureProtocol` to `true` instead.

File: .github/workflows/gradlebuild.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@ jobs:
           settings-path: ${{ github.workspace }} # location for the settings.xml file
 
       - name: Build with Gradle
-        run: gradle build
+        run: gradle shadowRemapJar
```
Comment (ImperatorStorm) [inline .github/workflows/gradlebuild.yml:21]: reasoning behind this?

Comment (IMS212) [reply .github/workflows/gradlebuild.yml:21]: gradlew build caused some unexplainable license errors even though the license header is the same (and was automatically generated by the same program on top of that!) Also, the README states to use this.

Comment (ImperatorStorm) [reply .github/workflows/gradlebuild.yml:21]: fair enough

**Comment by LambdAurora** [comment]
Done in 54090a1.

**Pull request closed without merging**